components/libc/libdl/dlelf.c

```diff
@@ -38,7 +40,7 @@ rt_err_t dlmodule_load_shared_object(struct rt_dlmodule* module, void *module_pt
         if (phdr[index].p_type != PT_LOAD)
             continue;
 
-        LOG_D("LOAD segment: %d, 0x%p, 0x%08x", index, phdr[index].p_vaddr, phdr[index].p_memsz);
+        //LOG_D("LOAD segment: %d, 0x%p, 0x%08x", index, phdr[index].p_vaddr, phdr[index].p_memsz);
```

Comment:
why remove this log?

Reply:
> It's better to write code as,
> 
> ```c
> #ifdef _64bit
> typedef rt_uint64_t Elf_Addr;
> #else
> typedef rt_uint32_t Elf_Addr;
> #endif
> ```
@BernardXiong  It really is a better way.